### PR TITLE
Cleanup Disk Scan Service

### PR DESCRIPTION
**Pull request by Qstick**
Cleanup Disk Scan Service

#### Database Migration
NO

#### Description
Remove `GetVideoFiles`
Rename `GetMusicFiles` to `GetAudioFiles`
Fix Tests

#### Todos
- [x] Tests

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

File: src/NzbDrone.Core.Test/MediaFiles/DiskScanServiceTests/ScanFixture.cs [hunk 1]
```diff
@@ -273,15 +272,15 @@ public void should_not_scan_subfolders_that_start_with_period()
...
                        {
-                           Path.Combine(_artist.Path, "Season 1", ".@__thumb", "file1.mkv").AsOsAgnostic(),
-                           Path.Combine(_artist.Path, "Season 1", ".@__THUMB", "file2.mkv").AsOsAgnostic(),
-                           Path.Combine(_artist.Path, "Season 1", ".hidden", "file2.mkv").AsOsAgnostic(),
-                           Path.Combine(_artist.Path, "Season 1", ".AppleDouble", "s01e01.mkv").AsOsAgnostic(),
-                           Path.Combine(_artist.Path, "Season 1", "s01e01.mkv").AsOsAgnostic()
+                           Path.Combine(_artist.Path, "Season 1", ".@__thumb", "file1.flac").AsOsAgnostic(),
+                           Path.Combine(_artist.Path, "Season 1", ".@__THUMB", "file2.flac").AsOsAgnostic(),
+                           Path.Combine(_artist.Path, "Season 1", ".hidden", "file2.flac").AsOsAgnostic(),
+                           Path.Combine(_artist.Path, "Season 1", ".AppleDouble", "s01e01.flac").AsOsAgnostic(),
```
Comment (danielunderwood) [inline src/NzbDrone.Core.Test/MediaFiles/DiskScanServiceTests/ScanFixture.cs]: Just noticed this. Mac also likes to create files called `.DS_STORE`. Do we have anything to detect those files and handle accordingly.

File: src/NzbDrone.Core.Test/MediaFiles/DiskScanServiceTests/ScanFixture.cs [hunk 2]
```diff
@@ -346,12 +345,12 @@ public void should_scan_dotHack_folder()
         [Test]
         public void should_exclude_osx_metadata_files()
         {
-            GivenSeriesFolder();
+            GivenArtistFolder();
 
             GivenFiles(new List<string>
                        {
                            Path.Combine(_artist.Path, "._24 The Status Quo Combustion.mp4").AsOsAgnostic(),
```
Comment (danielunderwood) [inline src/NzbDrone.Core.Test/MediaFiles/DiskScanServiceTests/ScanFixture.cs]: Do mp4 files work as audio too or should this also be changed/removed?

Comment (Qstick) [reply src/NzbDrone.Core.Test/MediaFiles/DiskScanServiceTests/ScanFixture.cs]: Should be changed

File: src/NzbDrone.Core.Test/MediaFiles/DiskScanServiceTests/ScanFixture.cs (hunk 1 above)
Comment (Qstick) [reply src/NzbDrone.Core.Test/MediaFiles/DiskScanServiceTests/ScanFixture.cs]: added as a test case. Lidarr should ignore as its not a recognized audio or extras extension.

**Pull request merged**